CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.2.0] - 2020-08-13
+### Changed
+- k8s stateful's name is based on instance_name so we support more Redises in one project
```

Comment:
Maybe:
```
Change K8s stateful set's name to contain `instance_name` postfix to support more Redis instances in one GCP project
```

Reply:
You are only person using postfix and not suffix I've ever met :D Otherwise good idea, implemented